includes/course-theme/class-sensei-course-theme-templates.php

```diff
@@ -240,7 +238,11 @@ private function load_file_templates() {
 	 */
 	public function add_course_theme_block_templates( $templates, $query, $template_type ) {
 
-		if ( 'wp_template' !== $template_type ) {
+		if ( 'wp_template' !== $template_type || $query['wp_id'] ) {
```

Comment:
I think this line is generating a warning when there's no 'wp_id' in the query var. This happens when I load a lesson on the frontend:
```
[30-Sep-2022 11:54:40 UTC] PHP Notice:  Undefined index: wp_id in /Users/***/Documents/work/A8C/Projects/sensei/includes/course-theme/class-sensei-course-theme-templates.php on line 241
[30-Sep-2022 11:54:40 UTC] PHP Stack trace:
```

Reply:
Ah sorry I'll fix it in another PR.

Reply:
Raised https://github.com/Automattic/sensei/pull/5828 thank you for raising this @onubrooks